projects/mac_ai/testing/prepare_release.py

```
@@ -0,0 +1,227 @@
+import os
+import pathlib
+import logging
+import tarfile
+
+from projects.core.library import env, config, run, configure_logging, export
+from projects.matrix_benchmarking.library import visualize
+
+import prepare_llama_cpp, utils, remote_access, podman_machine, brew, podman, prepare_virglrenderer
+import llama_cpp, ollama, ramalama
+
+def create_remoting_tarball(base_work_dir):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)